configure.ac

```diff
@@ -336,12 +336,6 @@ then
       AC_SUBST([LIBQATNAME], "libqatengine")
       CFLAGS_BACK="${CFLAGS}"
```

Comment:
Line 337 and 338 is not needed as it was added as a part of pkey structure detection